### PR TITLE
Allow rejecting subscriptions from the channel

### PR DESCRIPTION
**Pull request by lifo**

### CODE REVIEW

**Comment by kayakyakr** [comment]
This is awesome. Was thinking that this was missing.

**Comment by rafaelfranca** [comment]
The feature looks good to me but it lacks documentation.

**Comment by lifo** [comment]
It's WIP. I'm not done yet 😊

**Comment by rafaelfranca** [comment]
:+1:

File: lib/action_cable/channel/base.rb [hunk 1]
```diff
@@ -183,6 +198,14 @@ def subscription_confirmation_sent?
           @subscription_confirmation_sent
         end
 
+        def reject!
```
Comment (dhh) [inline lib/action_cable/channel/base.rb]: IMO, we should just use `#reject`. There's not a non-bang version of this method.

Comment (lifo) [inline lib/action_cable/channel/base.rb]: Yeah, makes sense. Changed in 37fe48928e4d4b254791eb8034c20baabe06a483

**Pull request merged**